<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conversationForm</class>
 <widget class="QWidget" name="conversationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0">
   <item>
    <layout class="QVBoxLayout" name="conversationLayout">
     <item>
      <widget class="QLabel" name="conversationLabel">
       <property name="text">
        <string>Conversation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="conversationDisplay"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="busyIndicator">
     <property name="visible">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;img src=&quot;assets/ai-spark.png&quot; width=&quot;16&quot; height=&quot;16&quot;&gt;&lt;span style=&quot;line-height:16px;&quot;&gt; Thinking... &lt;/span&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileLayout">
     <item>
      <widget class="QListWidget" name="attachedFilesList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="fileButtonsLayout">
       <item>
        <widget class="QPushButton" name="attachDirectoryButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; padding-left: 10px</string>
         </property>
         <property name="text">
          <string>Attach Directory</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="attachButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; padding-left: 10px</string>
         </property>
         <property name="text">
          <string>Attach Files</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>12</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="messageLayout">
     <item>
      <widget class="QTextEdit" name="userInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="placeholderText">
        <string>Type your message here...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="sendButtonLayout">
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left; padding-left: 10px</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
